docs/specification.md

```diff
@@ -523,7 +523,6 @@ Parameter (in Michelson):
 - Fails with `NOT_PROPOSING_STAGE` if the current stage is not a proposing one.
 - Fails with `FAIL_PROPOSAL_CHECK` if the proposal is rejected by `proposal_check`
   from the configuration.
-- Fails with `MAX_PROPOSALS_REACHED` if the current amount of ongoing proposals is at max value set by the config.
```

Comment:
There's also a mention of `max_proposals` in the "Configuration" section that should be removed (line 95 right now).

Reply:
Removed this now.